server/src/main/java/org/opensearch/extensions/settings/ImplimentedInterfaceRequestHandler.java

```diff
@@ -0,0 +1,45 @@
+package org.opensearch.extensions.settings;
```

Comment:
I don't think we need a whole new registration handler for this (but I may be wrong).

OpenSearch already sends an "ExtensionInitRequest" to the Extension, which currently responds with an `InitializeExtensionResponse` containing just the name.  If we update that response to include this implemented extension enum list/set, that would be a reasonable point to add it to the DiscoveryExtensionNode.

Reply:
Yes understood, I will do it that way

Thanks